drivers/usb/device/usb_dc_stm32.c

```diff
@@ -472,33 +477,15 @@ static int usb_dc_stm32_init(void)
 	HAL_StatusTypeDef status;
 	int ret;
 	unsigned int i;
-
+	usb_dc_stm32_state.pcd.Instance = (PCD_TypeDef *)DT_INST_REG_ADDR(0);
```

Comment:
Let's use the macro defined earlier:
```suggestion
	usb_dc_stm32_state.pcd.Instance = (PCD_TypeDef *)USB_BASE_ADDRESS;
```
(Otherwise, it looks like `USB_BASE_ADDRESS` is unused)

Reply:
nitpicking: you forgot to remove `USB_BASE_ADDRESS` macro in this source file.